cmd/version.go

```diff
@@ -1,7 +1,7 @@
 package cmd
 
 import (
-	log "github.com/Sirupsen/logrus"
```

Comment:
His repo was renamed to be lowercase `sirupsen/logrus`. This is a problem with os's with case sensitive filesystems (like osx/linux)